cexec/main.go

```diff
@@ -1,111 +1,202 @@
 package main
 
 import (
+	"context"
+	"errors"
+	"flag"
 	"fmt"
+	"log/slog"
 	"os"
 	"os/exec"
+	"os/signal"
 	"path/filepath"
 	"strings"
 	"syscall"
 
-	log "github.com/sirupsen/logrus"
+	"github.com/peterbourgon/ff/v3"
 )
 
-const mountAction = "/mountAction"
+const (
+	mountAction = "/mountAction"
+	envPrefix   = "CEXEC"
+)
+
+type settings struct {
+	blockDevice        string
+	filesystemType     string
+	chroot             string
+	defaultInterpreter string
+	cmdLine            string
+	updateResolvConf   bool
+}
 
 func main() {
-	fmt.Printf("CEXEC - Chroot Exec\n------------------------\n")
+	ctx, done := signal.NotifyContext(context.Background(), os.Interrupt, syscall.SIGHUP, syscall.SIGTERM)
```

Comment:
```suggestion
	ctx, done := signal.NotifyContext(context.Background(), syscall.SIGINT, syscall.SIGHUP, syscall.SIGTERM)
```

Shall we just use `syscall` for consistency?

Reply:
sure thing.